src/Main.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import "./Main.css"
+import "../node_modules/bootstrap/dist/css/bootstrap.min.css"
+import "../node_modules/bootstrap/dist/js/bootstrap.min.js"
+import "../node_modules/jquery/src/jquery.js"
+import "../node_modules/popper.js/dist/popper.min.js"
+
+
+class Navbar extends React.Component {
```

Comment:
Great - now let's try to put those links in an overflow menu when the screen is mobile-sized! Check this out for more details:

https://getbootstrap.com/docs/4.0/components/navbar/

Reply:
I think it is utilizing bootstrap for the overflow menu already. Is this what you're referring to?
![image](https://user-images.githubusercontent.com/29843006/42062037-1a922fe2-7ae1-11e8-981b-01872197062b.png)

Reply:
Yep - usually when condensed, these links will collapse into a hamburger icon. See this:

http://static.jesusheredia.info/images/38/1.png

When you click on the icon, it should collapse/expand the menu.